modules/bidmyadzBidAdapter.js

```diff
@@ -0,0 +1,151 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, NATIVE, VIDEO } from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
+import {config} from '../src/config.js';
+
+const BIDDER_CODE = 'bidmyadz';
+const AD_URL = 'https://endpoint.bidmyadz.com/pbjs';
+const SYNC_URL = 'https://cookie-sync.bidmyadz.com';
+
+function isBidResponseValid (bid) {
```

Comment:
isBidResponseValid() function should catch this:
![image](https://user-images.githubusercontent.com/12955750/122932133-c4b4a080-d36d-11eb-9aed-cbc1b980b985.png)

Reply:
Added additional check.